libs/dal/src/repositories/integration/integration.entity.ts

```diff
@@ -59,4 +30,6 @@ export class IntegrationEntity {
   deletedBy: string;
 }
 
+export type ICredentialsEntity = ICredentials;
```

Comment:
was not sure what to call this one :/ 
because it is basically a type of interface, i left the I for now.
any thought about it?